src/mev/bolt-boost/bolt-boost-launcher.star

```diff
@@ -0,0 +1,117 @@
+input_parser = import_module("../../package_io/input_parser.star")
+static_files = import_module("../../static_files/static_files.star")
+
+FLASHBOTS_MEV_BOOST_PROTOCOL = "TCP"
+
+SERVICE_NAME="bolt-boost"
+BOLT_BOOST_CONFIG_FILENAME="cb-bolt-config.toml"
+BOLT_BOOST_CONFIG_MOUNT_DIRPATH_ON_SERVICE="/config"
+BOLT_BOOST_BUILDER_PROXY_PORT=18551
+
+USED_PORTS = {
+    "api": shared_utils.new_port_spec(
+        input_parser.FLASHBOTS_MEV_BOOST_PORT, FLASHBOTS_MEV_BOOST_PROTOCOL, wait="5s"
+    )
+}
+
+# The min/max CPU/memory that mev-boost can use
+MIN_CPU = 10
+MAX_CPU = 500
+MIN_MEMORY = 16
+MAX_MEMORY = 256
+
+
+def launch(
+    plan,
+    bolt_boost_image,
+    service_name,
+    relays_config,
+    bolt_sidecar_config,
+    network_params,
+    global_node_selectors,
+):
+    config = get_bolt_boost_config(
+        plan,
+        bolt_boost_image,
+        relays_config,
+        bolt_sidecar_config,
+        global_node_selectors,
+    )
+
+    mev_boost_service = plan.add_service(service_name, config)
+
+    return mev_boost_context_module.new_mev_boost_context(
+        mev_boost_service.ip_address, input_parser.FLASHBOTS_MEV_BOOST_PORT
+    )
+
+
+def get_bolt_boost_config(
+    plan,
+    image,
+    relays_config,
+    bolt_sidecar_config,
+    node_selectors,
+):
+    # Read the template file for Bolt Boost configuration
+    bolt_boost_config_template = read_file(
+        static_files.BOLT_BOOST_CONFIG_TEMPLATE_FILEPATH
+    )
+
+    # Generate the data to be used in the Bolt Boost configuration,
+    # wrap them together in a struct
+    bolt_boost_config_template_data = new_bolt_boost_config_template_data(image, relays_config, bolt_sidecar_config)
+    bolt_boost_config_template_and_data = shared_utils.new_template_and_data(
+        bolt_boost_config_template, bolt_boost_config_template_data
+    )
+
+    # Map the relative destination filepaths to the template/data pairs
+    template_and_data_by_rel_dest_filepath = {}
+    template_and_data_by_rel_dest_filepath[BOLT_BOOST_CONFIG_FILENAME] = bolt_boost_config_template_and_data
+
+    # Render the templates to files in the artifact directory
+    config_files_artifact_name = plan.render_templates(
+        template_and_data_by_rel_dest_filepath
+    )
+
+    return ServiceConfig(
+        image=image,
+        ports=USED_PORTS,
+        files={
+            BOLT_BOOST_CONFIG_MOUNT_DIRPATH_ON_SERVICE: config_files_artifact_name
+        },
+        env_vars={
+            "CB_CONFIG": shared_utils.path_join(
+                BOLT_BOOST_CONFIG_MOUNT_DIRPATH_ON_SERVICE,
+                BOLT_BOOST_CONFIG_FILENAME,
+            )
+        },
+        min_cpu=MIN_CPU,
+        max_cpu=MAX_CPU,
+        min_memory=MIN_MEMORY,
+        max_memory=MAX_MEMORY,
+        node_selectors=node_selectors,
+    )
+
+def new_bolt_boost_config_template_data(image, relays_config, bolt_sidecar_config):
+    return {
+        "chain": "Holesky",
```

Comment:
This probably doesn't change anything but can we use "Kurtosis"?

Reply:
Unfortunately not because the value "Kurtosis" is not supported, only "Mainnet", "Holesky" and "Helder". But I've seen that `Custom` is supported! https://github.com/commit-boost/commit-boost-client/blob/main/crates/common/src/types.rs#L21-L26 going go to change it

Reply:
For ease of life I'm going to address it another PR.